wordpress-mobile/WordPress-Android
# [Reader Customization] Update links style and color

> [!Warning]
> This should only be merged after https://github.com/wordpress-mobile/WordPress-Android/pull/20506

Update links style to:
- All schemes: normal weight, underlined
- Default scheme: link color is the current blue we have
- Other schemes: link color is the primary text color

Internal ref: p1711719108252369/1711551014.024749-slack-C06PGTY5PKK

https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/7ca025c2-1a00-4551-821a-e672b65edcef

-----

## To Test:

Since the feature is still in development, follow the initial instrucitons in the "To Test" section of https://github.com/wordpress-mobile/WordPress-Android/pull/20506

- Go to Reader
- Tap any post with links in the content body
- **Verify** the link color and decoration matches the expecations
- Tap the "Reading Preferences" top bar option
- Change the color scheme (test all if possible)
- **Verify** the link color and decoration matches the expecations for each selected color scheme

-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

3. What automated tests I added (or what prevented me from doing so)

    - N/A

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
